GOVERNANCE.md

```diff
@@ -0,0 +1,95 @@
+  Occasional private chat messages may be exchanged,
+  e.g. when setting up services that require passwords,
+  but otherwise all exchanges that impact the project either happen in issue or PR discussions,
+  or in the [Gitter chat room](https://gitter.im/tldr-pages/tldr)
+  (which is open to all, and publicly logged).
+
+- **All decisions are made by community consensus.**
+  This does not mean there has to be unanimity,
+  nor that decisions result from vote counts.
+  What it means is that every interested member of the community can voice their thoughts,
+  and different positions are ideally resolved via
+  [informed consent](https://en.wikipedia.org/wiki/Sociocracy#Consent_vs._consensus)
+  of the involved people, who accept the collective decision
+  as "good enough for now, safe enough to try".
+
+
+## II. Role transitions
+
+The main goal of these principles is to support a continuous replenishing of the management team
+via a **smooth transition flow between community roles** —
+from newcomer, to occasional contributor, to regular contributor, to maintainer.
+This way the project can adapt in a flexible way
+to the the natural variations in availability and interest of its contributors,
+ensuring long-term resilience,
+and avoiding [single points of failure](https://en.wikipedia.org/wiki/Bus_factor).
+
+To this end, rather than assigning roles and tasks to people,
+these guidelines instead aim to **recognize the work that people already do**.
+Everyone is therefore encouraged to get involved and contribute to the project in whatever way they prefer,
+and we will strive to **get barriers out of the way** of these contributions.
+
+To ensure that these processes are transparent, predictable, and free from subective judgment,
+the metrics used are simple, objective, and publicly available.
+
+- Regular contributors shall be recognized as collaborators in the organization.
+
+    - Specifically: once a contributor has had **5 pull requests merged**,
+      they should be invited to become a
+      [**member of the tldr-pages organization**](https://github.com/orgs/tldr-pages/people).
+      This means they will be able to push commits to all of the organization's repositories,
+      merge PRs, label and close issues, among other things.
+      Note: All members of the tldr-pages organization must make their membership public.
+
+- Members of the organization who demonstrate interest in performing maintainership tasks,
+  by reviewing and/or merging PRs, responding to and labeling issues, and generally doing project maintenance work,
+  shall be made part of the maintenance team, and their name added to the list of current maintainers
+  in the [MAINTAINERS.md](MAINTAINERS.md) file.
+
+    - Specifically: once a contributor has been an organization member for at least 3 months,
+      and has **reviewed or merged 10 pull requests** by other contributors,
```

Comment:
I would rather this be merged 10 pull requests and reviewed 5 pull requests. I want to keep a minimum bar on the reviewing part.

Reply:
Fine by me :+1:

Reply:
Probably worth including some hint at needing some substance to the review, otherwise you'll be drowned in "LGTM" comments, and newcomers will be surprised when maintainers disagree with the "LGTM" littered on their PRs.

Reply:
Good point. I guess phrasing it as "non-trivial reviews" will suffice, but let me know if you have other ideas.

Reply:
Good points here!